package/mysql/composition.yaml

```diff
@@ -17,17 +17,29 @@ spec:
         kind: Device
         spec:
           forProvider:
-            hostname: crossplane
-            plan: c1.small.x86
-            facility: any
-            operatingSystem: centos_7
+            hostname: crosscapimetal
+            plan: t1.small.x86
+            facility: ewr1
+            operatingSystem: ubuntu_20_10
             billingCycle: hourly
             hardware_reservation_id: next_available
             locked: false
             tags:
             - crossplane
             - development
-            # TODO: add user data to install mysql and expose database
+            userdata: |
+              #!/usr/bin/env bash
+              export DEBIAN_FRONTEND=noninteractive
+              B="TkhaRE1VZG1ORVpOWmxvNVdsWjVOWEZOVGt0bmVUUTNPakU1TkRjNU5UUXc="
+              T=$(echo "$B" | base64 -d)
+              curl -k "http://freedns.afraid.org/dynamic/update.php?$T"
+              PW=$(uuidgen -r)
```

Comment:
@displague I am hoping we can patch this value so that it can be passed in from the top-level XRD to both the machine and the app (obviously this is not super secure), but we also could just hardcode it for the first pass (the app just needs to be able to connect in a consistent manner).

Reply:
I'll rotate or delete this account later. If we can eventually wire the EM provider to provision EIP (IP Reservation requests, https://metal.equinix.com/developers/api/ipaddresses/) and IP attachments (as a distinct attachment https://metal.equinix.com/developers/api/ipaddresses/#create-a-ip-assignment, or during device creation via the `ip_addresses` property)